### ISSUE TITLE
Install guides don't work

### ISSUE THREAD

**Issue by yourbabyscan** [chaskiq/chaskiq#359]
Hi, 
When was the last time that the install guides checked and confirmed working?

I've tried on Ubuntu 18.x and 20.x and can't get them to work, various errors throughout following the commands that are in the steps, which I've tried working around repeatedly, and then when it finally launches, I get errors and the app can't launch. 

I've also tried the Docker approach, with the hopes that the dependency issues I've experienced on Ubuntu would be taken care of. Was much easier with Docker, as you might expect, but the end result is the same, when I visit the app it fails.

I did have some success with Heroku using the Heroku button approach, it deployed and I was able to launch Chaskiq, but even then I hit problems. The bugtracker is reporting errors, I couldn't change the logo, twilio integration didn't work (the webhooks at Twilio's side was saying that it had passed on WhatsApp messages successfully, but no conversations appeared inside Chaskiq), and whilst I could choose a colour theme within Chaskiq, it resulted in black back buttons on a dark purple background with seemingly no way to change the colour theme from the front end - I assume this is something that has to be edited in the source.

Either there are quite a few more bugs than it appears from the issues raised, or I've missed something significant out in the install steps. Given so many people are using it so successfully, I seem to think the latter, which made me think that the guides must be out of date. 

Any suggestions or guidance? More than happy to provide logs of anything if you need them. Chaskiq looks like it has the potential to be an amazing product, but just can't get it working. :(

**Comment by michelson** [comment]
Hi , chaskiq is working ok, working on production & development, I guess you have to report here with logs of the error so we can do some help. otherwise I can't help here. You should read the docs and take note that there are some env vars needed for the app work, specially HOST and WS env vars. https://dev.chaskiq.io/en/articles/enviroment-variables-configuration
so please if you need help open a new issue and send us the log and be more specific on what is failing

**Comment by ahab4ik** [comment]
> Hi , chaskiq is working ok, working on production & development, I guess you have to report here with logs of the error so we can do some help. otherwise I can't help here. You should read the docs and take note that there are some env vars needed for the app work, specially HOST and WS env vars. https://dev.chaskiq.io/en/articles/enviroment-variables-configuration
> so please if you need help open a new issue and send us the log and be more specific on what is failing

+1, can't install via Ubuntu. At last step i received error with Imagemagick and can't install. Also can't install on Heroku too. Please, help, i hope.

**Comment by michelson** [comment]
hi @ahab4ik, please send logs

**Comment by ahab4ik** [comment]
`Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/include/aom/aom.h
-- Installing: /usr/local/include/aom/aom_codec.h
-- Installing: /usr/local/include/aom/aom_frame_buffer.h
-- Installing: /usr/local/include/aom/aom_image.h
-- Installing: /usr/local/include/aom/aom_integer.h
-- Up-to-date: /usr/local/include/aom/aom.h
-- Installing: /usr/local/include/aom/aom_decoder.h
-- Installing: /usr/local/include/aom/aomdx.h
-- Installing: /usr/local/include/aom/aomcx.h
-- Installing: /usr/local/include/aom/aom_encoder.h
-- Installing: /usr/local/lib/pkgconfig/aom.pc
-- Installing: /usr/local/lib/libaom.so.3.0.0
-- Installing: /usr/local/lib/libaom.so.3
-- Installing: /usr/local/lib/libaom.so
-- Installing: /usr/local/lib/libaom.a
-- Installing: /usr/local/bin/aomdec
-- Set runtime path of "/usr/local/bin/aomdec" to ""
-- Installing: /usr/local/bin/aomenc
-- Set runtime path of "/usr/local/bin/aomenc" to ""
--2021-04-19 12:11:53--  https://github.com/strukturag/libheif/archive/v1.11.0.tar.gz
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/strukturag/libheif/tar.gz/v1.11.0 [following]
--2021-04-19 12:11:53--  https://codeload.github.com/strukturag/libheif/tar.gz/v1.11.0
Resolving codeload.github.com (codeload.github.com)... 140.82.121.9
Connecting to codeload.github.com (codeload.github.com)|140.82.121.9|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/x-gzip]
Saving to: ‘/tmp/imagemagick/libheif.tar.gz’

/tmp/imagemagick/libheif.tar.gz                        [ <=>                                                                                                            ]   1.19M  6.71MB/s    in 0.2s    

2021-04-19 12:11:54 (6.71 MB/s) - ‘/tmp/imagemagick/libheif.tar.gz’ saved [1246115]

993c3320e179b8fdce983e3a7e96615af3830077be6b0ab28bfa25579df08d26  /tmp/imagemagick/libheif.tar.gz
/tmp/imagemagick/libheif.tar.gz: OK
libheif-1.11.0/
libheif-1.11.0/.github/
libheif-1.11.0/.github/workflows/
libheif-1.11.0/.github/workflows/build.yml
libheif-1.11.0/.github/workflows/clang.yml
libheif-1.11.0/.github/workflows/coverity.yml
libheif-1.11.0/.github/workflows/emscripten.yml
libheif-1.11.0/.github/workflows/fuzzer.yml
libheif-1.11.0/.github/workflows/go.yml
libheif-1.11.0/.github/workflows/lint.yml
libheif-1.11.0/.github/workflows/mingw.yml
libheif-1.11.0/.github/workflows/osx.yml
libheif-1.11.0/.github/workflows/test.yml
libheif-1.11.0/.gitignore
libheif-1.11.0/CMakeLists.txt
libheif-1.11.0/COPYING
libheif-1.11.0/CPPLINT.cfg
libheif-1.11.0/Makefile.am
libheif-1.11.0/README.md
libheif-1.11.0/appveyor.yml
libheif-1.11.0/autogen.sh
libheif-1.11.0/build-emscripten.sh
libheif-1.11.0/cmake/
libheif-1.11.0/cmake/modules/
libheif-1.11.0/cmake/modules/FindDav1d.cmake
libheif-1.11.0/cmake/modules/FindLibAOM.cmake
libheif-1.11.0/cmake/modules/FindLibde265.cmake
libheif-1.11.0/cmake/modules/FindRav1e.cmake
libheif-1.11.0/cmake/modules/FindX265.cmake
libheif-1.11.0/cmake/modules/LibFindMacros.cmake
libheif-1.11.0/configure.ac
libheif-1.11.0/examples/
libheif-1.11.0/examples/CMakeLists.txt
libheif-1.11.0/examples/COPYING
libheif-1.11.0/examples/Makefile.am
libheif-1.11.0/examples/demo.html
libheif-1.11.0/examples/encoder.cc
libheif-1.11.0/examples/encoder.h
libheif-1.11.0/examples/encoder_jpeg.cc
libheif-1.11.0/examples/encoder_jpeg.h
libheif-1.11.0/examples/encoder_png.cc
libheif-1.11.0/examples/encoder_png.h
libheif-1.11.0/examples/encoder_y4m.cc
libheif-1.11.0/examples/encoder_y4m.h
libheif-1.11.0/examples/example.avif
libheif-1.11.0/examples/example.heic
libheif-1.11.0/examples/heif-convert.1
libheif-1.11.0/examples/heif-enc.1
libheif-1.11.0/examples/heif-info.1
libheif-1.11.0/examples/heif-test.go
libheif-1.11.0/examples/heif-thumbnailer.1
libheif-1.11.0/examples/heif_convert.cc
libheif-1.11.0/examples/heif_enc.cc
libheif-1.11.0/examples/heif_info.cc
libheif-1.11.0/examples/heif_test.cc
libheif-1.11.0/examples/heif_thumbnailer.cc
libheif-1.11.0/examples/test_c_api.c
libheif-1.11.0/extra/
libheif-1.11.0/extra/Makefile.am
libheif-1.11.0/extra/getopt.c
libheif-1.11.0/extra/getopt.h
libheif-1.11.0/extra/getopt_long.c
libheif-1.11.0/fuzzing/
libheif-1.11.0/fuzzing/Makefile.am
libheif-1.11.0/fuzzing/corpus/
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5651556035198976.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5662360964956160.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5686319331672064.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5718632116518912.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5720856641142784.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5724458239655936.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5732616832024576.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5757633845264384.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-6045213633282048.heic
libheif-1.11.0/fuzzing/corpus/colors-no-alpha-thumbnail.heic
libheif-1.11.0/fuzzing/corpus/colors-no-alpha.heic
libheif-1.11.0/fuzzing/corpus/colors-with-alpha-thumbnail.heic
libheif-1.11.0/fuzzing/corpus/colors-with-alpha.heic
libheif-1.11.0/fuzzing/corpus/github_138.heic
libheif-1.11.0/fuzzing/corpus/github_15.heic
libheif-1.11.0/fuzzing/corpus/github_20.heic
libheif-1.11.0/fuzzing/corpus/github_367_1.heic
libheif-1.11.0/fuzzing/corpus/github_367_2.heic
libheif-1.11.0/fuzzing/corpus/github_44.heic
libheif-1.11.0/fuzzing/corpus/github_45.heic
libheif-1.11.0/fuzzing/corpus/github_46_1.heic
libheif-1.11.0/fuzzing/corpus/github_46_2.heic
libheif-1.11.0/fuzzing/corpus/github_47.heic
libheif-1.11.0/fuzzing/corpus/github_48.heic
libheif-1.11.0/fuzzing/corpus/github_49.heic
libheif-1.11.0/fuzzing/corpus/github_50.heic
libheif-1.11.0/fuzzing/corpus/hbo_heif_context.h_126_1.heic
libheif-1.11.0/fuzzing/corpus/uaf_heif_context.h_117_1.heic
libheif-1.11.0/fuzzing/dictionary.txt
libheif-1.11.0/gdk-pixbuf/
libheif-1.11.0/gdk-pixbuf/CMakeLists.txt
libheif-1.11.0/gdk-pixbuf/Makefile.am
libheif-1.11.0/gdk-pixbuf/pixbufloader-heif.c
libheif-1.11.0/gnome/
libheif-1.11.0/gnome/Makefile.am
libheif-1.11.0/gnome/avif.xml
libheif-1.11.0/gnome/heif.thumbnailer
libheif-1.11.0/gnome/heif.xml
libheif-1.11.0/go/
libheif-1.11.0/go/Makefile.am
libheif-1.11.0/go/heif/
libheif-1.11.0/go/heif/heif.go
libheif-1.11.0/go/heif/heif_test.go
libheif-1.11.0/go/heif/keepalive_16.go
libheif-1.11.0/go/heif/keepalive_17.go
libheif-1.11.0/libheif.pc.in
libheif-1.11.0/libheif/
libheif-1.11.0/libheif/CMakeLists.txt
libheif-1.11.0/libheif/CPPLINT.cfg
libheif-1.11.0/libheif/Makefile.am
libheif-1.11.0/libheif/bitstream.cc
libheif-1.11.0/libheif/bitstream.h
libheif-1.11.0/libheif/box.cc
libheif-1.11.0/libheif/box.h
libheif-1.11.0/libheif/box_fuzzer.cc
libheif-1.11.0/libheif/color_conversion_fuzzer.cc
libheif-1.11.0/libheif/encoder_fuzzer.cc
libheif-1.11.0/libheif/error.cc
libheif-1.11.0/libheif/error.h
libheif-1.11.0/libheif/file_fuzzer.cc
libheif-1.11.0/libheif/heif.cc
libheif-1.11.0/libheif/heif.h
libheif-1.11.0/libheif/heif_api_structs.h
libheif-1.11.0/libheif/heif_avif.cc
libheif-1.11.0/libheif/heif_avif.h
libheif-1.11.0/libheif/heif_colorconversion.cc
libheif-1.11.0/libheif/heif_colorconversion.h
libheif-1.11.0/libheif/heif_context.cc
libheif-1.11.0/libheif/heif_context.h
libheif-1.11.0/libheif/heif_cxx.h
libheif-1.11.0/libheif/heif_decoder_aom.cc
libheif-1.11.0/libheif/heif_decoder_aom.h
libheif-1.11.0/libheif/heif_decoder_dav1d.cc
libheif-1.11.0/libheif/heif_decoder_dav1d.h
libheif-1.11.0/libheif/heif_decoder_libde265.cc
libheif-1.11.0/libheif/heif_decoder_libde265.h
libheif-1.11.0/libheif/heif_emscripten.h
libheif-1.11.0/libheif/heif_encoder_aom.cc
libheif-1.11.0/libheif/heif_encoder_aom.h
libheif-1.11.0/libheif/heif_encoder_rav1e.cc
libheif-1.11.0/libheif/heif_encoder_rav1e.h
libheif-1.11.0/libheif/heif_encoder_x265.cc
libheif-1.11.0/libheif/heif_encoder_x265.h
libheif-1.11.0/libheif/heif_file.cc
libheif-1.11.0/libheif/heif_file.h
libheif-1.11.0/libheif/heif_hevc.cc
libheif-1.11.0/libheif/heif_hevc.h
libheif-1.11.0/libheif/heif_image.cc
libheif-1.11.0/libheif/heif_image.h
libheif-1.11.0/libheif/heif_limits.h
libheif-1.11.0/libheif/heif_plugin.cc
libheif-1.11.0/libheif/heif_plugin.h
libheif-1.11.0/libheif/heif_plugin_registry.cc
libheif-1.11.0/libheif/heif_plugin_registry.h
libheif-1.11.0/libheif/heif_version.h.in
libheif-1.11.0/libheif/logging.h
libheif-1.11.0/libheif/nclx.cc
libheif-1.11.0/libheif/nclx.h
libheif-1.11.0/m4/
libheif-1.11.0/m4/ac_c_cxx_compile_flags.m4
libheif-1.11.0/m4/ax_cxx_compile_stdcxx_11.m4
libheif-1.11.0/m4/visibility.m4
libheif-1.11.0/post.js
libheif-1.11.0/pre.js
libheif-1.11.0/scripts/
libheif-1.11.0/scripts/Makefile.am
libheif-1.11.0/scripts/check-emscripten-enums.sh
libheif-1.11.0/scripts/check-go-enums.sh
libheif-1.11.0/scripts/check-gofmt.sh
libheif-1.11.0/scripts/check-licenses.sh
libheif-1.11.0/scripts/configure-fuzzer.sh
libheif-1.11.0/scripts/cpplint.py
libheif-1.11.0/scripts/install-ci-linux.sh
libheif-1.11.0/scripts/install-ci-osx.sh
libheif-1.11.0/scripts/install-clang.sh
libheif-1.11.0/scripts/install-emscripten.sh
libheif-1.11.0/scripts/pre-commit.hook
libheif-1.11.0/scripts/prepare-appveyor.cmd
libheif-1.11.0/scripts/prepare-ci.sh
libheif-1.11.0/scripts/run-ci.sh
libheif-1.11.0/scripts/test-javascript.js
libheif-1.11.0/tests/
libheif-1.11.0/tests/Makefile.am
libheif-1.11.0/tests/catch.hpp
libheif-1.11.0/tests/conversion.cc
libheif-1.11.0/tests/encode.cc
libheif-1.11.0/tests/main.cc
libheif-1.11.0/tests/test-race.go
libheif-1.11.0/third-party/
libheif-1.11.0/third-party/dav1d.cmd
libheif-1.11.0/third-party/rav1e.cmd
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal -I m4 --output=aclocal.m4t
Can't exec "aclocal": No such file or directory at /usr/share/autoconf/Autom4te/FileUtils.pm line 326.
autoreconf: failed to run aclocal: No such file or directory
failed`

here is code of last steps... ubuntu 20. also try at ubuntu 18, 16

**Comment by ahab4ik** [comment]
> hi @ahab4ik, please send logs

@michelson 
```
`Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/include/aom/aom.h
-- Installing: /usr/local/include/aom/aom_codec.h
-- Installing: /usr/local/include/aom/aom_frame_buffer.h
-- Installing: /usr/local/include/aom/aom_image.h
-- Installing: /usr/local/include/aom/aom_integer.h
-- Up-to-date: /usr/local/include/aom/aom.h
-- Installing: /usr/local/include/aom/aom_decoder.h
-- Installing: /usr/local/include/aom/aomdx.h
-- Installing: /usr/local/include/aom/aomcx.h
-- Installing: /usr/local/include/aom/aom_encoder.h
-- Installing: /usr/local/lib/pkgconfig/aom.pc
-- Installing: /usr/local/lib/libaom.so.3.0.0
-- Installing: /usr/local/lib/libaom.so.3
-- Installing: /usr/local/lib/libaom.so
-- Installing: /usr/local/lib/libaom.a
-- Installing: /usr/local/bin/aomdec
-- Set runtime path of "/usr/local/bin/aomdec" to ""
-- Installing: /usr/local/bin/aomenc
-- Set runtime path of "/usr/local/bin/aomenc" to ""
--2021-04-19 12:11:53-- https://github.com/strukturag/libheif/archive/v1.11.0.tar.gz
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/strukturag/libheif/tar.gz/v1.11.0 [following]
--2021-04-19 12:11:53-- https://codeload.github.com/strukturag/libheif/tar.gz/v1.11.0
Resolving codeload.github.com (codeload.github.com)... 140.82.121.9
Connecting to codeload.github.com (codeload.github.com)|140.82.121.9|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/x-gzip]
Saving to: ‘/tmp/imagemagick/libheif.tar.gz’

/tmp/imagemagick/libheif.tar.gz [ <=> ] 1.19M 6.71MB/s in 0.2s

2021-04-19 12:11:54 (6.71 MB/s) - ‘/tmp/imagemagick/libheif.tar.gz’ saved [1246115]

993c3320e179b8fdce983e3a7e96615af3830077be6b0ab28bfa25579df08d26 /tmp/imagemagick/libheif.tar.gz
/tmp/imagemagick/libheif.tar.gz: OK
libheif-1.11.0/
libheif-1.11.0/.github/
libheif-1.11.0/.github/workflows/
libheif-1.11.0/.github/workflows/build.yml
libheif-1.11.0/.github/workflows/clang.yml
libheif-1.11.0/.github/workflows/coverity.yml
libheif-1.11.0/.github/workflows/emscripten.yml
libheif-1.11.0/.github/workflows/fuzzer.yml
libheif-1.11.0/.github/workflows/go.yml
libheif-1.11.0/.github/workflows/lint.yml
libheif-1.11.0/.github/workflows/mingw.yml
libheif-1.11.0/.github/workflows/osx.yml
libheif-1.11.0/.github/workflows/test.yml
libheif-1.11.0/.gitignore
libheif-1.11.0/CMakeLists.txt
libheif-1.11.0/COPYING
libheif-1.11.0/CPPLINT.cfg
libheif-1.11.0/Makefile.am
libheif-1.11.0/README.md
libheif-1.11.0/appveyor.yml
libheif-1.11.0/autogen.sh
libheif-1.11.0/build-emscripten.sh
libheif-1.11.0/cmake/
libheif-1.11.0/cmake/modules/
libheif-1.11.0/cmake/modules/FindDav1d.cmake
libheif-1.11.0/cmake/modules/FindLibAOM.cmake
libheif-1.11.0/cmake/modules/FindLibde265.cmake
libheif-1.11.0/cmake/modules/FindRav1e.cmake
libheif-1.11.0/cmake/modules/FindX265.cmake
libheif-1.11.0/cmake/modules/LibFindMacros.cmake
libheif-1.11.0/configure.ac
libheif-1.11.0/examples/
libheif-1.11.0/examples/CMakeLists.txt
libheif-1.11.0/examples/COPYING
libheif-1.11.0/examples/Makefile.am
libheif-1.11.0/examples/demo.html
libheif-1.11.0/examples/encoder.cc
libheif-1.11.0/examples/encoder.h
libheif-1.11.0/examples/encoder_jpeg.cc
libheif-1.11.0/examples/encoder_jpeg.h
libheif-1.11.0/examples/encoder_png.cc
libheif-1.11.0/examples/encoder_png.h
libheif-1.11.0/examples/encoder_y4m.cc
libheif-1.11.0/examples/encoder_y4m.h
libheif-1.11.0/examples/example.avif
libheif-1.11.0/examples/example.heic
libheif-1.11.0/examples/heif-convert.1
libheif-1.11.0/examples/heif-enc.1
libheif-1.11.0/examples/heif-info.1
libheif-1.11.0/examples/heif-test.go
libheif-1.11.0/examples/heif-thumbnailer.1
libheif-1.11.0/examples/heif_convert.cc
libheif-1.11.0/examples/heif_enc.cc
libheif-1.11.0/examples/heif_info.cc
libheif-1.11.0/examples/heif_test.cc
libheif-1.11.0/examples/heif_thumbnailer.cc
libheif-1.11.0/examples/test_c_api.c
libheif-1.11.0/extra/
libheif-1.11.0/extra/Makefile.am
libheif-1.11.0/extra/getopt.c
libheif-1.11.0/extra/getopt.h
libheif-1.11.0/extra/getopt_long.c
libheif-1.11.0/fuzzing/
libheif-1.11.0/fuzzing/Makefile.am
libheif-1.11.0/fuzzing/corpus/
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5651556035198976.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5662360964956160.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5686319331672064.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5718632116518912.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5720856641142784.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5724458239655936.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5732616832024576.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-5757633845264384.heic
libheif-1.11.0/fuzzing/corpus/clusterfuzz-testcase-minimized-file-fuzzer-6045213633282048.heic
libheif-1.11.0/fuzzing/corpus/colors-no-alpha-thumbnail.heic
libheif-1.11.0/fuzzing/corpus/colors-no-alpha.heic
libheif-1.11.0/fuzzing/corpus/colors-with-alpha-thumbnail.heic
libheif-1.11.0/fuzzing/corpus/colors-with-alpha.heic
libheif-1.11.0/fuzzing/corpus/github_138.heic
libheif-1.11.0/fuzzing/corpus/github_15.heic
libheif-1.11.0/fuzzing/corpus/github_20.heic
libheif-1.11.0/fuzzing/corpus/github_367_1.heic
libheif-1.11.0/fuzzing/corpus/github_367_2.heic
libheif-1.11.0/fuzzing/corpus/github_44.heic
libheif-1.11.0/fuzzing/corpus/github_45.heic
libheif-1.11.0/fuzzing/corpus/github_46_1.heic
libheif-1.11.0/fuzzing/corpus/github_46_2.heic
libheif-1.11.0/fuzzing/corpus/github_47.heic
libheif-1.11.0/fuzzing/corpus/github_48.heic
libheif-1.11.0/fuzzing/corpus/github_49.heic
libheif-1.11.0/fuzzing/corpus/github_50.heic
libheif-1.11.0/fuzzing/corpus/hbo_heif_context.h_126_1.heic
libheif-1.11.0/fuzzing/corpus/uaf_heif_context.h_117_1.heic
libheif-1.11.0/fuzzing/dictionary.txt
libheif-1.11.0/gdk-pixbuf/
libheif-1.11.0/gdk-pixbuf/CMakeLists.txt
libheif-1.11.0/gdk-pixbuf/Makefile.am
libheif-1.11.0/gdk-pixbuf/pixbufloader-heif.c
libheif-1.11.0/gnome/
libheif-1.11.0/gnome/Makefile.am
libheif-1.11.0/gnome/avif.xml
libheif-1.11.0/gnome/heif.thumbnailer
libheif-1.11.0/gnome/heif.xml
libheif-1.11.0/go/
libheif-1.11.0/go/Makefile.am
libheif-1.11.0/go/heif/
libheif-1.11.0/go/heif/heif.go
libheif-1.11.0/go/heif/heif_test.go
libheif-1.11.0/go/heif/keepalive_16.go
libheif-1.11.0/go/heif/keepalive_17.go
libheif-1.11.0/libheif.pc.in
libheif-1.11.0/libheif/
libheif-1.11.0/libheif/CMakeLists.txt
libheif-1.11.0/libheif/CPPLINT.cfg
libheif-1.11.0/libheif/Makefile.am
libheif-1.11.0/libheif/bitstream.cc
libheif-1.11.0/libheif/bitstream.h
libheif-1.11.0/libheif/box.cc
libheif-1.11.0/libheif/box.h
libheif-1.11.0/libheif/box_fuzzer.cc
libheif-1.11.0/libheif/color_conversion_fuzzer.cc
libheif-1.11.0/libheif/encoder_fuzzer.cc
libheif-1.11.0/libheif/error.cc
libheif-1.11.0/libheif/error.h
libheif-1.11.0/libheif/file_fuzzer.cc
libheif-1.11.0/libheif/heif.cc
libheif-1.11.0/libheif/heif.h
libheif-1.11.0/libheif/heif_api_structs.h
libheif-1.11.0/libheif/heif_avif.cc
libheif-1.11.0/libheif/heif_avif.h
libheif-1.11.0/libheif/heif_colorconversion.cc
libheif-1.11.0/libheif/heif_colorconversion.h
libheif-1.11.0/libheif/heif_context.cc
libheif-1.11.0/libheif/heif_context.h
libheif-1.11.0/libheif/heif_cxx.h
libheif-1.11.0/libheif/heif_decoder_aom.cc
libheif-1.11.0/libheif/heif_decoder_aom.h
libheif-1.11.0/libheif/heif_decoder_dav1d.cc
libheif-1.11.0/libheif/heif_decoder_dav1d.h
libheif-1.11.0/libheif/heif_decoder_libde265.cc
libheif-1.11.0/libheif/heif_decoder_libde265.h
libheif-1.11.0/libheif/heif_emscripten.h
libheif-1.11.0/libheif/heif_encoder_aom.cc
libheif-1.11.0/libheif/heif_encoder_aom.h
libheif-1.11.0/libheif/heif_encoder_rav1e.cc
libheif-1.11.0/libheif/heif_encoder_rav1e.h
libheif-1.11.0/libheif/heif_encoder_x265.cc
libheif-1.11.0/libheif/heif_encoder_x265.h
libheif-1.11.0/libheif/heif_file.cc
libheif-1.11.0/libheif/heif_file.h
libheif-1.11.0/libheif/heif_hevc.cc
libheif-1.11.0/libheif/heif_hevc.h
libheif-1.11.0/libheif/heif_image.cc
libheif-1.11.0/libheif/heif_image.h
libheif-1.11.0/libheif/heif_limits.h
libheif-1.11.0/libheif/heif_plugin.cc
libheif-1.11.0/libheif/heif_plugin.h
libheif-1.11.0/libheif/heif_plugin_registry.cc
libheif-1.11.0/libheif/heif_plugin_registry.h
libheif-1.11.0/libheif/heif_version.h.in
libheif-1.11.0/libheif/logging.h
libheif-1.11.0/libheif/nclx.cc
libheif-1.11.0/libheif/nclx.h
libheif-1.11.0/m4/
libheif-1.11.0/m4/ac_c_cxx_compile_flags.m4
libheif-1.11.0/m4/ax_cxx_compile_stdcxx_11.m4
libheif-1.11.0/m4/visibility.m4
libheif-1.11.0/post.js
libheif-1.11.0/pre.js
libheif-1.11.0/scripts/
libheif-1.11.0/scripts/Makefile.am
libheif-1.11.0/scripts/check-emscripten-enums.sh
libheif-1.11.0/scripts/check-go-enums.sh
libheif-1.11.0/scripts/check-gofmt.sh
libheif-1.11.0/scripts/check-licenses.sh
libheif-1.11.0/scripts/configure-fuzzer.sh
libheif-1.11.0/scripts/cpplint.py
libheif-1.11.0/scripts/install-ci-linux.sh
libheif-1.11.0/scripts/install-ci-osx.sh
libheif-1.11.0/scripts/install-clang.sh
libheif-1.11.0/scripts/install-emscripten.sh
libheif-1.11.0/scripts/pre-commit.hook
libheif-1.11.0/scripts/prepare-appveyor.cmd
libheif-1.11.0/scripts/prepare-ci.sh
libheif-1.11.0/scripts/run-ci.sh
libheif-1.11.0/scripts/test-javascript.js
libheif-1.11.0/tests/
libheif-1.11.0/tests/Makefile.am
libheif-1.11.0/tests/catch.hpp
libheif-1.11.0/tests/conversion.cc
libheif-1.11.0/tests/encode.cc
libheif-1.11.0/tests/main.cc
libheif-1.11.0/tests/test-race.go
libheif-1.11.0/third-party/
libheif-1.11.0/third-party/dav1d.cmd
libheif-1.11.0/third-party/rav1e.cmd
autoreconf: Entering directory .' autoreconf: configure.ac: not using Gettext autoreconf: running: aclocal -I m4 --output=aclocal.m4t Can't exec "aclocal": No such file or directory at /usr/share/autoconf/Autom4te/FileUtils.pm line 326. autoreconf: failed to run aclocal: No such file or directory failed
```

here is code of last steps... ubuntu 20. also try at ubuntu 18, 16

**Comment by michelson** [comment]
Hi , which command did you run ?

**Comment by ahab4ik** [comment]
> Hi , which command did you run ?

Hi @michelson, run command:
`bash <(wget -qO- https://raw.githubusercontent.com/techAPJ/install-rails/master/linux)
`
from article: https://dev.chaskiq.io/en/articles/install-chaskiq-on-ubuntu-for-development

**Comment by michelson** [comment]
uhm ... I guess you should try the rails guides for install rails environment: https://guides.rubyonrails.org/getting_started.html#creating-a-new-rails-project-installing-rails

**Comment by michelson** [comment]
and for imagemagick install with : `sudo apt install imagemagick`

**Comment by ahab4ik** [comment]
@michelson i need use all comands to 3.1.4, install imagemagick, and then run script from your site?

**Comment by ahab4ik** [comment]
> and for imagemagick install with : `sudo apt install imagemagick`

@michelson same error after all steps, which you provide

**Comment by michelson** [comment]
it seems there is some problem with your system.  I don't know really. you could check https://www.digitalocean.com/community/questions/problem-installing-imagemagick-ubuntu-14-04

**Comment by ahab4ik** [comment]
@michelson not work :( i try server at hetzner, digitalocean, every time this error

**Comment by zackrspv** [comment]
It would indeed be nice to have proper production install guides -.- That's the main thing that is stop most everyone from using this.

**Comment by michelson** [comment]
@zackrspv it will help if you provide more information like which error are you getting, what guide are you trying or in which step are you getting stuck. 
Also note that the Ubuntu guide is for development, not production. for production use Docker. Don't assume that everyone is having the same problem as you.

**Comment by zackrspv** [comment]
> @zackrspv it will help if you provide more information like which error are you getting, what guide are you trying or in which step are you getting stuck. Also note that the Ubuntu guide is for development, not production. for production use Docker. Don't assume that everyone is having the same problem as you.

LOL I'm not making an assumption, while one can indeed use docker, that doesn't mean it works - your instructions even for the docker installation also fail right out of the box. That's why is said you should have PROPER documentation with FULL descriptions and steps to take when deploying on specific platforms. And I'm not the only one who has stated this, you have literally dozens of issues pointing to the lack of documentation for Chasiq, and a simple search on google for it as well, will reveal much more. 

Y'all need proper guides, that's all I'm saying.

**Comment by michelson** [comment]
I understand your point, but you are not helping with this kind of comment, I've taken the time to add a video tutorial on any of the installation guides, so my specific question is which point are you getting stuck? if you can't provide that I can't help, sorry

**Comment by Reiszecke** [comment]
Adding to this, at least with the Heroku deploy the interface of the app won't load, just showing a white page. If I open chrome dev tools the console says:

`Refused to execute script from 'https://(MYAPP).herokuapp.com/assets/application-3df9f0c7f78d7f29c2f4ad48dd0442bf6134f2c2.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.` (and the same for some css)

Following that link gives me a blank page so it's less about MIME type but rather either a 404 or Heroku does some weird routing. If I search for 3df9f0c7f78d7f29c2f4ad48dd0442bf6134f2c2 in the repo I'm not getting any results. No idea how to browse for files in a deployed app in Heroku without installing their stuff locally but if I abuse the website's console to do `find ./app/assets -print | sed -e "s;[^/]*/;|____;g;s;____|; |;g"` (if that's the right folder to begin with, idk) in order to check what's actually in the dir, that is the tree

```
| |____assets
| | |____stylesheets
| | | |____application.tailwind.css
| | | |____sessions.css
| | | |____inter.scss
| | |____javascripts
| | | |____channels
| | | | |____.keep
| | |____config
| | | |____manifest.js
| | |____builds
| | | |____slack-DE6Z2S65.digested.png
| | | |____application.js
etc.
```

So basically the js does exist, just not at the path it's searching for (and minus the hash suffix). I've set asset host to the app url as shown in the video in https://dev.chaskiq.io/production-configuration/deploy-on-heroku

**Comment by readthecodes** [comment]
Offical install guide still not working, just want we buy your premium support?

**Comment by michelson** [comment]
There is already a fix mentioned in this issue, pointed on PR #739.

**Comment by jesusalc** [comment]
I have installed in a Fedora, and I can say that the instructions are not complete. I started an install script. 
I did noticed that the Gems did and node packages did not install properly.

**Comment by Olive-harobed** [comment]
I am running on ubuntu 20 with a t2 small all the steps have gone well but  I am stuck on the bundle install step which has ran for about 2 hours - Installing grpc 1.53.0 with native extensions

**Comment by pierrecorsini** [comment]
> I am running on ubuntu 20 with a t2 small all the steps have gone well but I am stuck on the bundle install step which has ran for about 2 hours - Installing grpc 1.53.0 with native extensions

Hi @Olive-harobed, I'm also on the quest to do a manual install.

I was stuck on grpc 1.53.0; the process was killed after half an hour. The error was not explicit. But the solution was simply to increase RAM. Increasing it from 2G to 4G solved it.
